plugins/telemetry/telemetry.cpp

```diff
@@ -238,6 +238,7 @@ void Telemetry::flight_mode_async(flight_mode_callback_t callback)
 
 const char *Telemetry::flight_mode_str(FlightMode flight_mode)
 {
+    LogDebug() << "Flight mode val " << flight_mode;
```

Comment:
This debug line should probably eventually be removed, what do you think?

Reply:
We can remove it. Caller may print this. We should make this `std::string`

Reply:
Hm, good point, we might as well.